sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -88,6 +88,71 @@ func resourceSumologicMonitorsLibraryMonitor() *schema.Resource {
 				Optional: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"trigger": {
```

Comment:
This would be an explicit sub field of triggers if I understand this correctly? I don't think this would work with the current apis

Reply:
Thanks for reviewing this! 
So this actually turns out to not be a concern since we eventually translate resource data from here to JSON model classes, which are then marshaled automatically to form the JSON payload) when making infrastructure calls from Terraform. When translating `trigger` blocks of the resource model to JSON, we flatten out the fields into `TriggerCondition`.

Reply:
BTW, I am working on (yet) another, and a shorter version of this change. I realized that the `trigger` block is unnecessary here--we can put condition blocks directly under `triggers` like so
```
triggers {
  logs_outlier_condition {
   field = ...
   ...
  }
}
```
Will send it out for review shortly.